collections/_infographics/energy/electricity-czechia.md

```diff
@@ -3,15 +3,30 @@ layout:        infographic
 data-our:      "https://docs.google.com/spreadsheets/d/1q8LGAmpIOhdNUtljeOWxljy_FIE63THlnr3z9lsJTlI/edit?usp=sharing"
 data-orig:
   - [ "Energy Regulatory Office", "http://www.eru.cz/documents/10540/4580207/Rocni_zprava_provoz_ES_2018.pdf/" ]
   - [ "IPCC 2014: ANNEX III", "https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_annex-iii.pdf" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/elektrina-cr" %}
+## How to read this graph
+
+* The graph below shows the balance of the Czech electricity sector in 2018. Looking from the left, you can see the share of individual sources in electricity production, to which we have assigned the corresponding amount of CO<sub>2</sub>eq emissions. Imports of electricity from abroad also contribute to the overall balance on the production side. Similarly, the consumption side, shown to the right of the overall balance, is further divided into electricity exports abroad, losses and net consumption, broken down by sector of the national economy.
+* At first glance, the graph shows the disproportion between production and associated emissions for coal-fired and thermal power plants. The latter account for 47% of electricity generation but account for 88% of all emissions in the electricity sector. If we add up the production of coal and gas-fired power plants, we arrive at 54% of the production from fossil sources, but they are also responsible for 96% of the sector's emissions. __The fossil part of the Czech electricity sector thus represents by far the largest source of greenhouse gas emissions in the Czech Republic across all sectors__, see the [related infographic](/infographics/emissions-czechia).
+
+* Another disparity can be seen between the amount of electricity imported from abroad (11.6 TWh) and the electricity exported to neighbouring countries (25.5 TWh) in 2018. The resulting __net export (13.9 TWh) is comparable to household consumption (15.1 TWh) and exceeds the consumption of the tertiary sector, including transport (13.2 TWh)__. The net export is also higher than, for example, the annual production at the two largest lignite-fired power plants in the country. The corresponding emissions (using the average coefficient for the Czech energy mix, see the methodology described below) correspond to approximately half of the emissions from passenger car transport in the Czech Republic. Moreover, net electricity exports have been increasing again since 2015<!--(see page 32, [ERO: Annual Report on the Operation of the EC of the Czech Republic 2018](http://www.eru.cz/documents/10540/4580207/Rocni_zprava_provoz_ES_2018.pdf/))-->). __The Czech Republic is generally one of the leading electricity exporters in the world, with Czech exports outpacing, for example, those of the People's Republic of China, which is many times larger__ (see [data for 2016](https://www.indexmundi.com/g/r.aspx?v=82) or [data for 2018](http://www.worldstopexports.com/electricity-exports-country/)).
```

Comment:
I'm unable to find the English reference for http://www.eru.cz/documents/10540/4580207/Rocni_zprava_provoz_ES_2018.pdf. Not sure if there is one available or an alternative?